doc-Installing_on_OpenShift_Container_Platform/topics/post-installation.adoc

```diff
@@ -145,6 +145,112 @@ $ oc create -f cfme-restore-job.yaml
 
 After the restore job is complete, you can scale the pods back up.
 
+
+
+[[updating-pods]]
+=== Updating {product-title_short}
+
+To apply package updates to a {product-title_short} appliance running on OpenShift Container Platform, delete the pods and redeploy them using updated {product-title_short} container images. To update your environment with a new template, follow the steps in xref:modified-template-update[].
+
+ifdef::cfme[]
+See the https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html/release_notes/index#technical_notes[Technical Notes] in the _Release Notes_ for a list of updated {product-title_short} container images and errata.
+endif::cfme[]
+
+
+[[image-update]]
+==== Updating the {product-title_short} Container Images
+
+To deploy new container images for {product-title_short}, delete the pods, update the images, then redeploy the updated pods:
+
+. Scale the application StatefulSets to 0 replicas:
```

Comment:
Hey Dayle,

Is this command run on the OCP server? Do we need a command instructing users to SSH to te OCP server first, just to foolproof things?

Reply:
Also should StatefulSets have any markup around it if its a config file parameter? (I'm assuming that's what it is)

Reply:
Hmm, good points here... it's true in other docs we might start each procedure with the SSH command (or mention it) for completeness. However, everything in this guide (except the last step of 'Logging into CloudForms") is done from OpenShift to administer CloudForms, and I believe OCP admins and regular users should know to run the "oc" commands from the OpenShift server. 

I'm really not sure whether we should mark up StatefulSets (it's an OpenShift object --> a set of pods). We haven't in a previous section, and I wonder if marking it up creates a situation where we need to mark up all the OpenShift terms, like template, pods, etc? (the capitalization makes it look weirder than the other terms)

I lean towards leaving the above two things as is, but what do you think?

Reply:
I think those explanations clear up any questions I had. The capitalization of StatefulSets did make me immediate think it was a config parameter, hence asking about markup. Thanks!